translate/cloud-client/snippets.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python
+
```

Comment:
Added a basic one. Most of our samples do not have READMEs and instead lean on the docs. I want to do something similar to what Jason has done for Node, but it's low on the priority list.